webproject/src/main/java/com/google/sps/authentication/AuthenticationHandlerSupplier.java

```diff
@@ -0,0 +1,11 @@
+package com.google.sps.authentication;
+
+public class AuthenticationHandlerSupplier {
```

Comment:
why not change AuthenticationHandler as injectable directly?

Reply:
if you need Supplier like this, it is possible to use Provider<> which Guice provide

Reply:
I have made the AuthenticationHandler injectable